### PR TITLE
Fix ES proxy to handle additional attribute

### PR DESCRIPTION
**Pull request by feng-tao**
### Summary of Changes

It turns out that ES response have more attribute than the one defined in model. Create a base class for all ES response model and include a method to contain all instance attributes.

### Tests

Unit test

### Documentation

_What documentation did you add or modify and why? Add any relevant links then remove this line_

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes. 
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does
- [x] PR passes `make test`
- [x] I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

**Comment by feng-tao** [comment]
test on staging and make sure everything works as expected(table, user, category search)

**Comment by feng-tao** [comment]
test on staging and make sure everything works as expected(table, user, category search)

**Comment by feng-tao** [comment]
test on staging and make sure everything works as expected(table, user, category search)

**Comment by feng-tao** [comment]
cc @youngyjd

File: search_service/api/table.py [hunk 1]
```diff
@@ -99,11 +99,11 @@ def get(self, *, field_name: str,
 
         try:
             results = self.proxy.fetch_table_search_results_with_field(
-                query_term=args.get('query_term'),
+                query_term=args.get('query_term', ''),
                 field_name=field_name,
                 field_value=field_value,
                 page_index=args['page_index'],
-                index=args['index']
+                index=args.get('index')
```
Comment (youngyjd) [inline search_service/api/table.py]: In what case this will make a difference?

Comment (feng-tao) [reply search_service/api/table.py]: if index doesn't exist in args, the previous one will throw exception.

**Pull request merged**